hypercube/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN --mount=type=cache,id=hypercube-apk,sharing=locked,from=cache,target=/var/ca
     addgroup nginx jwt && \
     cleanup.sh
 
-ENV HYPERCUBE_FCREPO_URL=fcrepo/fcrepo/rest \
+ENV HYPERCUBE_FCREPO_URL=fcrepo:8008/fcrepo/rest \
```

Comment:
:facepalm:

Reply:
I've taken the liberty to straight push that fix: https://github.com/Islandora-Devops/isle-buildkit/commit/b3e273a78ddbb7b1d8b052399299380372a2ec02

Sorry for the derp.

Reply:
Wouldn't break it either way, since it's not used for anything other than the configuration parser expects some value to be present. Nonetheless best that it's correct :)